### ISSUE TITLE
"Minimal length" of linear symbols

### ISSUE THREAD

**Issue by Symbian9** [OpenOrienteering/mapper#651]
You know that some linear ISOM symbols have "minimal length" rule, mapper also has option but it do nothing at this moment.
Is it possible automatically enlarge linear element to this size if user try create smaller line.
For example, if minimal option of symbol is 3 mm and user create line with only 2 mm length, then it would be automaticaly enlarged to 3 mm.
•——*→•

**Comment by dg0yt** [comment]
I don't think it is that easy. Lines are sometimes used as parts of large constructs. Imagine a short line connecting two parallel lines. You cannot simple make this line longer. I even think the ISOM should give more examples how to actually handle certain "minimal" situations.

The minimum length option produce a hint in the "Measure lenghts and areas" tool. But this window is not updated while drawing (#595).

**Comment by jan-gerard** [comment]
Maybe this could be solved by adding a feature that generates a list of warning/incompliances with the mapping guidelines. Automated behavior may often be undesired. But triggering a human might be useful.

**Comment by dg0yt** [comment]
There are already open issues #35 Problem widget, #302 Map report and problem browser tool.
